app.py

```diff
@@ -19,8 +19,26 @@
 
 from api.models import *
 
-@app.route('/', methods=['GET'])
+@app.route('/')
```

Comment:
Interesting that this doesn't need to specify the types of methods allowed. I assume it allows all types of message (GET, POST etc) although it wouldn't do anything with it.